Resources/Prototypes/Entities/Objects/Weapons/Guns/turrets.yml

```
@@ -73,52 +73,81 @@
     - type: BallisticAmmoProvider
-      proto: CartridgeCaselessRifle
-      capacity: 500
+      proto: CartridgeLightRifle # This way it drops spent casings.
+      capacity: 200 # Not infinite ammo, it should run out of ammo in a more reasonable timeframe.
     - type: HTN
       rootTask:
         task: TurretCompound
       blackboard:
         RotateSpeed: !type:Single
-          3.141
+          1.571 # At most 1 second to react to a target directly behind it, instead of instantaneous tracking  
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)